Update README.md, also added RtlInitAnsiStringEx, RtlInitUnicodeStringEx

pappyN4 opened the pull request:
updates to readme from win-raid posts

-usb speed reporting patch
-SMART reporting patch for storport 6.1.7601.23403
-x64 usbd.sys import
-x64 WdfLdr8 coexist patch

pappyN4 committed:
Update README.md

storport SMART reporting
usb speed reporting

pappyN4 committed:
Update README.md

x64 usbd.sys import

pappyN4 committed:
Update README.md

pappyN4 committed:
Update README.md

x64 WdfLdr8 coexist patch

pappyN4 committed:
Update README.md

update storport KB# 6.1.7601.25735
update msahci KB# 6.1.7601.25735 / clarify x64 search CC CC CC 
update storahci KB# 6.2.9200.20652

pappyN4 committed:
Update README.md

add AMD SATA 1.2.1.402, Samsung NVMe 3.3.0.2003

pappyN4 committed:
Update README.md

MovAX0xDEAD commented:
Thanks for your patches, i will check and update project

pappyN4 committed:
Update README.md

add UASP 6.2.9200.16384

pappyN4 committed:
Update README.md

UASP deadlock patch

pappyN4 committed:
Update README.md

pappyN4 committed:
Update README.md

renamed WppRecor.sys for DOS8.3 length limit

pappyN4 committed:
Update ntoskrnl_redirects.h

added
RtlInitAnsiStringEx
RtlInitUnicodeStringEx

pappyN4 committed:
Update ntoskrn8.c

add
RtlInitAnsiStringEx_k8
RtlInitUnicodeStringEx_k8

pappyN4 commented:
added RtlInitAnsiStringEx, RtlInitUnicodeStringEx
exists in 2003 but not in XP, used by setupdd.sys
added while testing windows install issue with NVME/XP that does not occur with NVME/2003x86

the-r3dacted commented:
Could you add RtlDowncaseUnicodeChar to your fork? I need that function for a driver.

pappyN4 committed:
Update README.md

updated for msahci 26057

pappyN4 committed:
Update ntoskrn8.rc

updated file version to 5.1.2600.11 to account for addition of 
RtlInitAnsiStringEx, RtlInitUnicodeStringEx

pappyN4 committed:
Update README.md

new driver versions from win8

pappyN4 committed:
Update ntoskrnl_redirects.h v12

pappyN4 committed:
Update ntoskrn8.c v12

pappyN4 committed:
Update ntoskrn8.rc v12

pappyN4 committed:
Update ntoskrnl_redirects.h v13

add memcmp for win7 x86 (not needed for x64)
-needed for ksecdd.sys 6.2.9200.24115

pappyN4 committed:
Update ntoskrn8.rc v13

v13, added memcmp for win7 x86

pappyN4 committed:
Update ntoskrnl_redirects.h V14

v14, added PoFxSetDeviceIdleTimeout for RtsUer.sys 10.0.26100.31288 x64 testing

pappyN4 committed:
Update ntoskrn8.rc

v14, added PoFxSetDeviceIdleTimeout for win7

pappyN4 committed:
Update ntoskrn8.c V14

pappyN4 committed:
Update ntoskrnl_redirects.h V14

remove memcmp from win7x86
